irobot_edu_sdk/robot.py

```diff
@@ -297,7 +313,7 @@ def when_touched(self, condition: list[bool, bool, bool, bool], callback: Callab
         """Register when touch callback of type: async def fn(front_left: bool, front_right: bool, back_left: bool, back_right: bool)."""
         self._when_touched.append(Event(condition, callback))
 
-    def when_cliff_sensor(self, condition: list[bool], callback: Callable[[bool], Awaitable[None]]):
+    def when_cliff_sensor(self, condition: list[bool, bool, bool, bool], callback: Callable[[bool], Awaitable[None]]):
```

Comment:
This is consistent with the current library but makes me realize I may need to do something about type hinting for the case where different robots have different numbers of arguments (this comment is more for me than you 😜 )